### PR TITLE
automated DMN BPMN script

### PR DESCRIPTION
**Pull request by lewis150193**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-254

### Change description ###
Automated DMN BPMN

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by indvamshi — APPROVED** [review]
looks fine.

File: scripts/camunda-deployment.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+done
+
+for file in ${IA_TASK_DMNS_BPMNS_PATH}/*.bpmn ${IA_TASK_DMNS_BPMNS_PATH}/*.dmn
+do
+	if [ -f "$file" ]
+	then
+curl --header "Content-Type: multipart/form-data" "ServiceAuthorization: ${AUTHORIZATION}"\
+  --request POST \
+  --form data=@$file \
+  "http://camunda-bpm/engine-rest/deployment/create"
```
Comment (alexbaeza) [inline scripts/camunda-deployment.sh:30]: let's make the url be an environment variable.

**Comment by alexbaeza** [comment]
Well done looks great! 🎉

File: README.md [hunk 2]
```diff
@@ -134,3 +134,58 @@ To run any of the service, Ingress should be enabled
...
+fr-idm-575d89f957-fb6rl                      1/1     Running   0          5m22s
+idam-web-admin-957474868-l62q6               1/1     Running   0          4m34s
+idam-web-public-c8cf99759-s86g4              1/1     Running   0          4m15s
+service-auth-provider-api-5744c5f89b-9rtm5   1/1     Running   0          6m2s
+shared-db-76d8954d5c-24t2g                   1/1     Running   0          5m28s
+sidam-api-59b66bf4cb-d24j6                   1/1     Running   0          5m19s
+wiremock-59669584fc-xcxjw                    1/1     Running   0          74s
+xui-webapp-7485d8c499-htmq5                  1/1     Running   0          71s
+```
+
```
Comment (alexbaeza) [inline README.md:184]: Don't need this either

File: README.md [hunk 3]
```diff
@@ -134,3 +134,58 @@ To run any of the service, Ingress should be enabled
...
+## Automating BPMN/DMN diagrams
+
+Add the following variables to your bash profile:
+ * `WA_BPMNS_DMNS_PATH` - to the path where your WA BPMNS are stored. [Link to path here](https://github.com/hmcts/wa-standalone-task-bpmn/tree/master/src/main/resources)
```
Comment (alexbaeza) [inline README.md:188]: Don't need this

**Review by alexbaeza — APPROVED** [review]

**Pull request merged**